CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java

```diff
@@ -138,33 +138,39 @@ public PluginParameters createParameters() {
      */
     private void displaySummaryAlert(final int importedRows, final List<String> validFilenames, final List<String> invalidFilenames) {
         Platform.runLater(() -> {
-            boolean success = true;
             final StringBuilder sbHeader = new StringBuilder();
             final StringBuilder sbMessage = new StringBuilder();
 
+            String fileFiles = (validFilenames.size() > 1) ? "files" : "file";
+            if (validFilenames.size() == 0) {
+                fileFiles = "files";
+            }
```

Comment:
Can be simplified to `final String fileFiles = (validFilenames.size() == 1) ? "file" : "files";`